docker-compose.yml

```diff
@@ -52,6 +52,9 @@ services:
       - mysql
       - redis
     env_file: .dev.env
+    environment:
+      - IMPACT_API_URL=${IMPACT_API_URL}
+      - IMPACT_API_ACCESS_TOKEN=${IMPACT_API_ACCESS_TOKEN}
```

Comment:
Created AC-5099 to handle switching to oauth/token rather than an explicit environment variable to token that will expire.